images/prow-tests/Dockerfile

```diff
@@ -100,20 +103,23 @@ ARG KUBETEST2_VERSION=ea3c260668075a29dd2b7f3ba4ddf1fe78bdf898
 ARG KIND_VERSION=v0.11.1
 ARG KO_VERSION=v0.8.2
 ARG PROTOC_GEN_GO_VERSION=v1.26.0
+ARG KIND_VERSION=v0.11.1
 
 # Extra tools through go get
 # These run using the golang image version of Go, not any defined by `gvm`
-RUN go get github.com/google/ko@${KO_VERSION}
-RUN go get github.com/boz/kail/cmd/kail
-RUN go get gotest.tools/gotestsum
-RUN go get github.com/raviqqe/liche@v0.0.0-20200229003944-f57a5d1c5be4  # stable liche version for checking md links
-RUN go get sigs.k8s.io/kind@${KIND_VERSION}
-RUN go get sigs.k8s.io/kubetest2@${KUBETEST2_VERSION}
-RUN go get sigs.k8s.io/kubetest2/kubetest2-gke@${KUBETEST2_VERSION}
-RUN go get sigs.k8s.io/kubetest2/kubetest2-kind@${KUBETEST2_VERSION}
-RUN go get sigs.k8s.io/kubetest2/kubetest2-tester-exec@${KUBETEST2_VERSION}
-RUN go get google.golang.org/protobuf/cmd/protoc-gen-go@${PROTOC_GEN_GO_VERSION}
-RUN go get github.com/google/go-licenses
+RUN go install github.com/google/ko@${KO_VERSION}
+RUN go get github.com/boz/kail/cmd/kail@v0.15.0
```

Comment:
Since you are here, would you mind also moving these remaining hardcoded versions to be ARGs?

Reply:
Also it'll be clearer if all the ARG lines are at the beginning of this file.